main.tf

```diff
@@ -13,10 +13,17 @@ data "aws_vpc" "default" {
 
 data "aws_availability_zones" "available" {}
 
+resource "null_resource" "default" {
```

Comment:
I don't understand why this `null_resource` is necessary.

Reply:
We get the existing `igw` by datasource `data.aws_internet_gateway.default` in the file `main.tf`
If there is the resource `igw` is not available then `terraform apply` is failed with error:
```![image](https://user-images.githubusercontent.com/1134449/30314598-dc0e3b46-97a9-11e7-99d8-e58413e0b952.png)